charts/catalog/templates/controller-manager-deployment.yaml

```diff
@@ -42,6 +42,10 @@ spec:
         {{- end }}
         - -v
         - "{{ .Values.controllerManager.verbosity }}"
+        {{- if .Values.controllerManager.resyncInterval }}
```

Comment:
Nit: This doesn't _have_ to be conditional since `values.yaml` is providing a default value.